.github/actions/release-maven/update-pom-version.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
```

Comment:
I'm not sure I follow the need for such a script: seems to me that the sed replacement could replace unrelated version that happen to match the regexp, while using the maven versions:set command would handle this more smoothly.
What am I missing?

Reply:
Before backporting the sed replacements herem, I tried to use the `mvn versions:set ` command, but it didn't work out of the box. I preferred to don't spend too much time on it as we can always improve that later on and the amount of work is already huge without it. 
One of the replacements that are not working out of the box with `mvn versions:set` is [this one](https://github.com/Activiti/activiti-cloud-application/blob/develop/activiti-cloud-dependencies/pom.xml#L8).

Reply:
Thanks for the explanation (and I'm not saying this should be done in the context of that PR): it makes sense that this version would not be replaced given the fact that this project is in another repository at https://github.com/Activiti/activiti-cloud/blob/develop/activiti-cloud-build/pom.xml#L7. This version should be handled as a dependency via a property in the root pom.
This maven management should definitely be reviewed as I see this as a big risk for a release to fail (or not represent correctly what we're releasing): this sed replacement might not be reliable.

Reply:
I agree, I don't like `sed` here neither. I don't think you can reference the parent version as a property, but it should be possible to review the module's inheritance.